clang/include/clang/Basic/Attr.td

```diff
@@ -1978,6 +1978,13 @@ def IntelFPGALocalStaticSlaveMemVar : SubsetSubject<Var,
          S->hasLocalStorage())}],
          "local variables, static variables, slave memory arguments">;
 
+def IntelFPGALocalStaticAgentMemVar : SubsetSubject<Var,
```

Comment:
You should also remove the definition of `IntelFPGALocalStaticSlaveMemVar ` as there are no users of the subject any longer.